elastic/kibana
# [Synthetics] adjust the status for project monitor cards

## Summary

Resolves https://github.com/elastic/kibana/issues/148673

Adjusts the status for project monitors to ensure they are rendered in the correct color.

<img width="1509" alt="Screen Shot 2023-01-10 at 3 10 38 PM" src="https://user-images.githubusercontent.com/11356435/211652040-40b85aa3-83f4-4293-ad7e-65dfa356dc13.png">

### Testing 
1. Create a project monitor. Wait a minute or two for it to execute against the service
2. Navigate to the Overview page
3. Ensure the project monitor card is displayed in the correct color, red for up and green for down
